library/Ivoz/Provider/Infrastructure/Api/Hostname/GetHostname.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Ivoz\Provider\Infrastructure\Api\Hostname;
```

Comment:
I would put this out of Api namespace. I don't see any relationship between "get host name" and API